clean-ci-resources.sh

```diff
@@ -99,4 +104,10 @@ for resource in 'volume snapshot' 'volume' 'floating ip' 'security group' 'keypa
 			./stale.sh -q $resource | report $resource | xargs --verbose --no-run-if-empty openstack $resource delete
 			;;
 	esac
+	CLEANUP_RC=$((CLEANUP_RC + $?))
 done
+set -e
+if [ $CLEANUP_RC != 1 ]; then
```

Comment:
Don't you need to check against the initial value for CLEANUP_RC instead?
```suggestion
if [ $CLEANUP_RC != 0 ]; then
```